### DISCUSSION TITLE
Using TMS coil

### DISCUSSION THREAD

**Discussion by BlackPencil-C** [simnibs/simnibs#130] [8. Simulation workflows] [answered]
I am interested in using my own coil design in SIMNIBS. I have seen the flowchart for converting my coil into a .ccd file, but it is too brief and I am still not clear on how to do it. Is there a detailed process available? Or could you tell me what each column of data in the .ccd file represents?

**Comment by oulap** [comment]
We are in the progress of integrating coil modeling into simnibs, so if you check back in a month or so hopefully we will have better documentation and examples on how to create coil models.

**Reply by BlackPencil-C** [reply]
It has been over a month now. May I ask about the progress of the documentation or examples？

**Reply by TorgeW** [reply]
The work on this is still in progress. Updates on this can be expected soon.

**Reply by TorgeW** [reply]
We are working on this but as always, things take time. I will update here when we get closer to a release

**Reply by TorgeW** [reply, marked as answer]
@BlackPencil-C 

We released a pre-release which implements our new coil format.
https://github.com/simnibs/simnibs/releases/tag/dev

Examples on how to use it can be found here:
https://github.com/simnibs/simnibs/tree/master/simnibs/examples/coils
